documentation/source/users/rmg/installation/ubuntuSubsystem.rst

```diff
@@ -0,0 +1,43 @@
+.. _ubuntuSubsystem:
+
+*****************************************************
+Installing RMG in the Ubuntu Subsystem on Windows 10
+*****************************************************
+
+Installing the Ubuntu Subsystem
+===================================
+
+1. Follow the instructions provided by Microsoft to install the Ubuntu subsystem, available `here
+<https://docs.microsoft.com/en-us/windows/wsl/install-win10>`_. The basic steps include enabling the Windows Linux
+subsystem from a powershell **run as an administrator**, and downloading the latest LTS version of Ubuntu
+*from the Windows store*. We highly recommend Ubuntu over the other Linux distros, as Ubuntu is used by many of the
+RMG developers.
+
+2. Once the Ubuntu subsystem is installed, open an Ubuntu terminal. Open a web browser in Windows and go to the
```

Comment:
Maybe direct them to first copy the link, then open a terminal to paste it?
Also, is it worth noting one of the ways of opening a terminal in Ubuntu?

Reply:
Good suggestions, I have made this more clear